test/support/assets/seeders/20111117063700-seedPerson.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+var nodeify = require('nodeify');
+
+module.exports = {
+  up: function (migration, DataTypes, done) {
+    nodeify(migration.bulkInsert('Person', [{
```

Comment:
IIRC it contains `nodify` to support the old versions of sequelize (the CLI basically supports all version of sequelize).